services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -29,6 +30,10 @@ export const ReviewSubmit = ({
     >(undefined)
     const history = useHistory()
     const modalRef = useRef<ModalRef>(null)
+    const { loggedInUser } = useAuth()
+
+    // pull the programs off the user
+    const statePrograms = loggedInUser && 'state' in loggedInUser && loggedInUser.state.programs || []
```

Comment:
Is there a case where a user at this page does not have a `state`?  If that happened, what error handling do we need?

Reply:
This page is only shown to StateUsers so I think we’re fine. But since this page and SubmissionSummary are both calling fetchSubmission2 we probably could do the same thing for both.

Reply:
To clarify don't all `role=STATE_USER` already for sure have a `state`? Or is there some case we need error handling around where a state user does not have a state?

Reply:
no they all should. I probably should have used an isStateUser() function. I think I’m going to put up a little cleanup PR